easybuild/easyconfigs/c/crossguide/crossguide-20190529-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,54 @@
+# Contribution from Imperial College London, UK
+# uploaded by J. Sassmannshausen
+
+easyblock = 'CMakeMake'
+
+name = 'crossguide'
```

Comment:
Isn't the package name `crossguid`?

Reply:
Yes, you are right. 'Brain-auto-correction' I guess. I changed both occurrences and also in PR #15228 and updated PR #16212 and #15245 accordingly.